tests/test_schema.py

```diff
@@ -1150,12 +1194,10 @@ class World(Base):
 
         world_schema = SQLAlchemySchemaNode(World)
 
-        self.assertTrue(
-            isinstance(world_schema['not_a_number'].typ, colander.String))
+        self.assertIsInstance(
```

Comment:
eh...  but Travis seems to have had no issue with this test in 2.6...  The docs are pretty clear that it was added in 2.7, though.